select.nix

```diff
@@ -56,6 +56,45 @@ rec {
     else
       throw "Expected a list or an attrset";
 
+  /**
+    Selector parser that supports parsing quoted attributes.
+
+    parseQuotedSelector :: str -> [str]
+
+    # Inputs
+
+    `x`
+
+    : 1\. Selector string (Specialised attribute path)
+
+    # Examples
+    :::{.example}
+
+    ```nix
+    parseQuotedSelector "foo.bar"
+    => [ "foo" "bar" ]
+    parseQuotedSelector ''someAttrset."foo.bar"''
+    => [ "someAttrset" "foo.bar" ]
+
+    ```
+  **/
+  parseQuotedSelector =
```

Comment:
This is not needed as a separate library function, it can be internal to `parseSelector`